UndertaleModCli/Program.cs

```diff
@@ -731,8 +731,8 @@ private void RunCSharpFile(string path)
     /// Evaluates and executes given C# code.
     /// </summary>
     /// <param name="code">The C# string to execute</param>
-    /// <param name="scriptFile">The path to the script file where <see cref="code"/> was executed from.
-    /// Leave as null, if it wasn't executed from a script file</param>
+    /// <param name="scriptFile">The path to the script file where the code was loaded from.
```

Comment:
I'd much rather use something like paramref instead of getting rid of it entirely.
(at least iirc paramref should work?)